Toolset/palettes/behaviors/revinspectorbehavior.livecodescript

```diff
@@ -176,8 +176,17 @@ private function inspectorCalculateLayout pGroupList, pLeft, pTop, pRowWidth, pS
    return tInspectorHeight
 end inspectorCalculateLayout
 
-local sManualHeight
+local sManualHeightOfStack, sNewHeight, sOldHeight
 on inspectorLayoutGroups pGroupList, pForceHeight
+   
+   # store height of stack when manually resizing
+   if pForceHeight <> true and the mouse is down and sNewHeight <> sOldHeight then
```

Comment:
Could you use `not ...` for `<> true` and `is not` for `<>` please? Also down needs to be quoted in `the mouse is "down"` to make sure this compiles in strict compilation mode.